scripts/check_rfcs/check_rfcs.spec.js

```diff
@@ -57,39 +50,46 @@ describe("checkRfcs script", () => {
     nock.cleanAll();
   });
 
-  describe("when not being run in CI", () => {
-    it("should run and call console log", async () => {
-      jest.resetModules();
-      const OLD_ENV = process.env.NODE_ENV;
-      process.env.NODE_ENV = "test";
-      await checkRfcs();
+  it("should not run the script when being run in CI", async () => {
+    const OLD_ENV = process.env.NODE_ENV;
+
+    ci.isCI = true;
+    process.env.NODE_ENV = "NOT-TEST";
+    await checkRfcs();
+
+    expect(consoleLogMock).not.toHaveBeenCalled();
+    process.env.NODE_ENV = OLD_ENV;
+  });
 
-      expect(consoleLogMock).toHaveBeenCalled();
+  describe("when not being run in CI", () => {
+    it("if script retrieves pull request data, should run and call console log once", async () => {
```

Comment:
```suggestion
    it("should run and call console log once when script retrieves pull request data, ", async () => {
```
Super minor and not expecting you to change this as we do this quite a lot in the codebase but the `it` should make contextual sense in my opinion.